src/commands/build.rs

```diff
@@ -19,7 +20,27 @@ pub fn build(cache: &Cache, project_type: &ProjectType) -> Result<(), failure::E
 
             commands::run(command, &command_name)?;
         }
+        ProjectType::Webpack => {
+            let wasm_pack_path =
+                install::install("wasm-pack", "rustwasm", cache)?.binary("wasm-pack")?;
+
+            wranglerjs::run_npm_install().expect("could not run `npm install`");
```

Comment:
I think  that this assume that the user has `npm` installed and available globally. Should we mention this on the readme or what will be the strategy for updating the docs once this PR gets merged?

Reply:
In order to use it the user must have `node` and `npm` available, see #82

Reply:
we need this to check that node and npm are present. see how we check using the `which` crate here: https://github.com/cloudflare/wrangler/blob/85f09e0d7b12971a224ba683fcd230ef31509cf4/src/install/mod.rs#L10

instead of install node and npm we'll need to panic and tell the user they need to install node and npm OR to run with whatever we decide is the way to override to the ProjectType::JavaScript

Reply:
i've added this functionality.